plugins/logger.py

```diff
@@ -12,7 +12,8 @@ class DiscordLogger(BasePlugin):
                 "message_delete",
                 "message_edit",
                 "member_join",
-                "member_remove"
+                "member_remove",
+                "log_event"
```

Comment:
The whole idea is for a cross-plugin support. Otherwise, it's going to require plugin-specific settings.